### PR TITLE
Update alpine to 3.14.0

### PR DESCRIPTION
**Pull request by little9**
This upgrades the alpine base image to 3.14.0. In the update, the references to the nginx config directory needed to be changed to the new location. We also needed to manually add an apk that provides the `preloadable_libiconv.so` for `php-iconv`

### CODE REVIEW

**Comment by little9** [comment]
@birkland @bseeger Do you have any thoughts about why this may be failing? Does the export test that is failing (20-export-tests.sh) look like an actual failure?

**Comment by bseeger** [comment]
Hi @little9  I'll take a peak, since that's code I implemented.  I'll try to look in the next hour or two.

**Comment by bseeger** [comment]
About 3 months ago we tackled a problem with PHP and it's iconv version.  @birkland added some libraries here and it made the issue go away: https://github.com/jhu-idc/idc-isle-buildkit/commit/1f66fa1d5aefc4f1e70c82560efabc5e60f9af03

What you're running into now is probably some sort of issue with the `iconv` in the new alpine image.  So, the idc code itself is fine, but this is what happens when the conversion tools in the image aren't working as expected.  

Truncated names in the citations strings are a direct result of `iconv` not behaving as expected: 

````     
 <div class="csl-entry">Edward Weston 2 Preferred Name Suffix, and
      Ansel Easton Adams 1 Preferred Name Suffix. 2020. “oo nimal ew
      itle”. 1. Knoxville Zoo.
      https://islandora-idc.traefik.me/node/40.
</div>
```

The `oo nimal new itle` should really be `Zoo Animal New Title`

**Comment by little9** [comment]
@bseeger Thank you for looking at this! I'll look into `iconv` in this newer verison of Alpine.

**Comment by little9** [comment]
@bseeger @birkland Do either of you mind if I merge this? I'd like to get the 3.14 update in before the 3.15 update.

File: abuild/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+FROM alpine:3.14
+
 
 RUN --mount=type=cache,target=/var/cache/apk \ 
     --mount=type=cache,target=/etc/cache/apk \
     ln -s /var/cache/apk /etc/apk/cache && \
     apk add --update \
         alpine-sdk \
         coreutils \
+        sudo \
```
Comment (bseeger) [inline abuild/Dockerfile:11]: I'm slightly curious as to why sudo is added?   When we log in we are root, so I'm wondering what value this adds or issue this fixes?

File: base/Dockerfile [hunk 2]
```diff
@@ -19,6 +19,7 @@ RUN --mount=type=cache,target=/var/cache/apk \
         netcat-openbsd \
         openssl \
         postgresql-client \
+        sudo \
```
Comment (bseeger) [inline base/Dockerfile:22]: same comment as the same above about sudo being added.

**Review by bseeger — COMMENTED** [review]
Looks good overall, just curious about the sudo thing.

File: abuild/Dockerfile (hunk 1 above)
Comment (little9) [reply abuild/Dockerfile:11]: @bseeger The ImageMagick build was failing before I added this. I'll remove it and run the build again to see the exact failure.

Comment (little9) [reply abuild/Dockerfile:11]: Here's one of the test runs for the upgrade without `sudo`: https://github.com/jhu-idc/idc-isle-buildkit/runs/4412701426?check_suite_focus=true

Comment (bseeger) [reply abuild/Dockerfile:11]: Weird, apparently Alpine doesn't generally ship with `sudo`, so I wonder how things worked before.

Comment (bseeger) [reply abuild/Dockerfile:11]: Thanks for looking at that.

**Review by bseeger — APPROVED** [review]
Looks good to me, providing the tests fail. Thanks, @little9.

**Pull request merged**